src/Model/MailCryptKeyPair.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Model;
+
+/**
+ * @author doobry <doobry@systemli.org>
+ */
+class MailCryptKeyPair
+{
+    /**
+     * @var string
```

Comment:
Also create as Trait for this properties. Keep in mind that the annotation does not match the getter (could be null)